content/03.results.md

```diff
@@ -42,78 +42,78 @@ Moreover, the OpenPBTA data does not feature methylation arrays which are now co
-| Low-grade astrocytic tumor                    | LGG, BRAF V600E                          | 53   |
-| Low-grade astrocytic tumor                    | LGG, BRAF V600E, CDKN2A/B                | 10   |
-| Low-grade astrocytic tumor                    | LGG, FGFR                                | 16   |
-| Low-grade astrocytic tumor                    | LGG, IDH                                 | 6    |
-| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF                       | 222  |
-| Low-grade astrocytic tumor                    | LGG, KIAA1549-BRAF, other MAPK           | 2    |
-| Low-grade astrocytic tumor                    | LGG, MYB/MYBL1                           | 4    |
-| Low-grade astrocytic tumor                    | LGG, NF1-germline                        | 12   |
-| Low-grade astrocytic tumor                    | LGG, NF1-germline, CDKN2A/B              | 2    |
-| Low-grade astrocytic tumor                    | LGG, NF1-germline, FGFR                  | 4    |
-| Low-grade astrocytic tumor                    | LGG, NF1-somatic                         | 4    |
-| Low-grade astrocytic tumor                    | LGG, NF1-somatic, FGFR                   | 2    |
-| Low-grade astrocytic tumor                    | LGG, NF1-somatic, NF1-germline, CDKN2A/B | 2    |
-| Low-grade astrocytic tumor                    | LGG, other MAPK                          | 23   |
-| Low-grade astrocytic tumor                    | LGG, RTK                                 | 22   |
-| Low-grade astrocytic tumor                    | LGG, RTK, CDKN2A/B                       | 2    |
-| Low-grade astrocytic tumor                    | LGG, wildtype                            | 84   |
-| Low-grade astrocytic tumor                    | SEGA, wildtype                           | 6    |
-| Mesenchymal non-meningothelial tumor          | EWS                                      | 21   |
-| Neuronal and mixed neuronal-glial tumor       | CNC                                      | 4    |
-| Neuronal and mixed neuronal-glial tumor       | EVN                                      | 2    |
-| Neuronal and mixed neuronal-glial tumor       | GNT, BRAF V600E                          | 2    |
-| Neuronal and mixed neuronal-glial tumor       | GNT, KIAA1549-BRAF                       | 4    |
-| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK                          | 2    |
-| Neuronal and mixed neuronal-glial tumor       | GNT, other MAPK, FGFR                    | 2    |
-| Neuronal and mixed neuronal-glial tumor       | GNT, RTK                                 | 4    |
-| Tumors of sellar region                       | CRANIO, ADAM                             | 51   |
-|                                               | Total                                    | 1281 |
+| Broad histology group                              | OpenPBTA molecular subtype                        | n    |
```

Comment:
@sjspielman @jashapiro what do you think of this wording? "Broad histology group"?

Reply:
Seems fine to me.